testdata/firebase_config.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
If a developer has the environment variable set to this before they run all tests, do the tests work, or will this break some of them?

Reply:
We overwrite the param before each of the options tests and reinstate it after.
I've tested it locally trying both a valid and an bad filename. the tests work either way, did I miss something?

Reply:
What I meant was that setting this env var will affect the options used by other tests (for other go admin packages) that might depend on the assumption that the config defaults to empty. But it's probably fine.

Reply:
What I meant was that setting this env var will affect the options used by other tests (for other go admin packages) that might depend on the assumption that the config defaults to empty. But it's probably fine.

Reply:
You are right, my test was wrong, if I add the variable locally, it fails those tests that that exist and do not set it. If guarded against this by wrapping the set and restore in the TestMain of each package, I will add this to the style guide. (i.e. when checking the integration tests, you should run them with and without the setting of the env vars)